src/java.base/share/classes/javax/crypto/EncryptedPrivateKeyInfo.java

```diff
@@ -181,7 +184,22 @@ public EncryptedPrivateKeyInfo(AlgorithmParameters algParams,
         if (algParams == null) {
             throw new NullPointerException("algParams must be non-null");
         }
-        this.algid = AlgorithmId.get(algParams);
+        AlgorithmId tmp;
```

Comment:
style-comment: meaningful variable names should be preferred (ie- choose something other than `tmp`)